baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java

```diff
@@ -94,10 +93,12 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
             return Description.NO_MATCH;
         }
 
-        SuggestedFix.Builder fix = SuggestedFix.builder();
-        String logSafeQualifiedClassName = SuggestedFixes.qualifyType(state, fix, "com.palantir.logsafe.Preconditions");
+        SuggestedFix.Builder fix = SuggestedFix.builder()
+                .addImport("com.palantir.logsafe.Preconditions")
+                .removeImport("com.google.common.base.Preconditions");
+
         String logSafeMethodName = getLogSafeMethodName(ASTHelpers.getSymbol(tree));
-        String replacement = String.format("%s.%s", logSafeQualifiedClassName, logSafeMethodName);
+        String replacement = String.format("Preconditions.%s", logSafeMethodName);
```

Comment:
This breaks compilation for files that use some replaceable guava preconditions, and others with interpolated parameters, which we cannot programmatically replace.

Reply:
Not sure what you mean by interpolated parameters?

Reply:
```java
com.google.common.base.Preconditions;

Preconditions.checkState(value != 5, "Value %s was not 5", value);
```